src/internal.c

```diff
@@ -24450,7 +24450,7 @@ static int DoServerKeyExchange(WOLFSSL* ssl, const byte* input,
             #else
                     enum wc_HashType hashType;
                     word16  verifySz;
-                    byte sigAlgo;
+                    byte sigAlgo = 0;
```

Comment:
For reference this just started due to https://github.com/wolfSSL/wolfssl/pull/4562 

Note: a `0` is `anonymous_sa_algo`, which I would suggest not using. You might consider using `0xFF` instead and I would get @SparkiDev to review, as it was his PR that added this additional variable / code.

Reply:
Thanks for the suggestion!!

Reply:
I think it is complaining that the hsType in DecodeSigAlg is not being set all the time now.
Please change DecodeSigAlg() to set a new enum value: invalid_sa_algo = 255
Also fix the formatting of the else in DecodeSigAlg()